config.sh

```diff
@@ -218,7 +218,9 @@ case "$1" in
 	echo - flame-kk
 	echo - rpi "(Revision B)"
 	echo - shinano
+	echo - shinano-l
 	echo - aries
+	echo - aries-l
```

Comment:
Nit: Maybe we should make it clear those are Z3 and Z3 Compact running KK or Lollipop ?